### PR TITLE
[12.0][FIX] mail_activity_done module

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
This PR intends to fix this issue: #379 

cc ~ @jbeficent @emagdalenaC2i

### CODE REVIEW

**Review by emagdalenaC2i — APPROVED** [review]
It works fine. Thanks

@ageficent Could you increment the version number?

**Comment by emagdalenaC2i** [comment]
@ageficent and @jbeficent Other non related issue in this module:

If you install this module it handle correctly the existing activities. 
But if you 
1) install the module, 
2) create a few activities and mark some of them as done

When you uninstall the module, that activities are no longer marked as done.

This can be an important issue if you have create and done thousand of activities.

**Comment by emagdalenaC2i** [comment]
And other issue. You can check this in runbot.
If you go to Contacts, and set the filter "Late activities" it will show all the late activities (done or not done)
image
As you can see this contact activities are done
image

**Comment by emagdalenaC2i** [comment]
Tested and code review
@ageficent Could you increment the version number?

**Comment by emagdalenaC2i** [comment]
@jbeficent are this PR ready to merge?

**Comment by emagdalenaC2i** [comment]
@jbeficent and @ageficent 
Please, could you finish this PR... and remember the issue reported by @aitorbouzas #386

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
@ageficent could you cherry-pick https://github.com/OCA/social/pull/395?

**Comment by AdriaGForgeFlow** [comment]
> @ageficent could you cherry-pick #395?

Done :+1:

File: mail_activity_done/migrations/11.0.2.0.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
```
Comment (MiquelRForgeFlow) [inline mail_activity_done/migrations/11.0.2.0.0/pre-migration.py]: change number version of the folder

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: mail_activity_done/views/mail_activity_views.xml [hunk 2]
```diff
@@ -27,19 +27,41 @@
...
+                <attribute name="domain">[('date_deadline', '=', context_today().strftime('%Y-%m-%d'))]</attribute>
             </filter>
             <filter name="activities_upcoming_all" position="attributes">
-                <attribute name="domain">[('date_deadline', '&gt;', context_today().strftime('%Y-%m-%d')), ('done', '!=', True)]</attribute>
+                <attribute name="domain">[('date_deadline', '&gt;', context_today().strftime('%Y-%m-%d'))]</attribute>
+            </filter>
+        </field>
+    </record>
+
+    <record id="res_partner_view_search_inherit_mail" model="ir.ui.view">
```
Comment (MiquelRForgeFlow) [inline mail_activity_done/views/mail_activity_views.xml:46]: apply in this view same changes as done above

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: mail_activity_done/migrations/11.0.2.0.0/pre-migration.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply mail_activity_done/migrations/11.0.2.0.0/pre-migration.py]: and move it to post-migration 😆

**Review by emagdalenaC2i — APPROVED** [review]
Review and tested in Runbot
You can merge once solve last comment of @mreficent about post-migration

**Review by MiquelRForgeFlow — APPROVED** [review]
:+1:

**Comment by emagdalenaC2i** [comment]
Are this ready to merge?

**Pull request merged**